getting-started/templates/systemlink-values.yaml

```diff
@@ -399,11 +403,14 @@ dashboardhost:
         ca_cert_path: "/etc/ssl/certs/dashboardhost/postgres-tls-certificate.pem"
 
       ## Plugin configuration
+      ## Uncomment to set Grafana plugin configuration.
       ##
-      plugins:
+      # plugins:
         ## Enter a comma-separated list of plugin identifiers to identify plugins that are allowed to be loaded even if they lack a valid signature.
+        ## Uncomment to set the list of unsigned plugins to load. This will override the plugins included by default so ensure you maintain
+        ## the list of plugins listed in the default values file.
         ##
-        allow_loading_unsigned_plugins: "ni-slnotebook-datasource"
+        # allow_loading_unsigned_plugins: ni-slnotebook-datasource,ni-sldataframe-datasource,ni-plotly-panel
```

Comment:
Because these Grafana plugins are such a core part of our experience I suggest we don't have these commented out by default. Its more of a corner case that a customer would want an SLE deployment without them than with them.

Reply:
This is the same confusion I had initially. These values are enabled by default in the Helm chart itself. Uncommenting this line will override the defaults so that the user can add their own plugins to the list. This is not an uncommon pattern with Helm.